tileserver/__init__.py

```diff
@@ -234,9 +234,10 @@ def async_store(store, tile_data, coord, format):
 def async_update_tiles_of_interest(redis_cache_index, coord):
     """update tiles of interest set"""
     try:
-        redis_cache_index.index_coord(coord)
+        if coord.zoom <= 20:
+            redis_cache_index.index_coord(coord)
         if coord.zoom > 18:
-            coord_at_z18 = coord.zoomTo(18)
+            coord_at_z18 = coord.zoomTo(18).container()
```

Comment:
Yup, that makes sense, thanks. But we definitely want it to be double-counting the stuff at 19 & 20? It wouldn't make more sense to have something like:

```python
if coord.zoom < 19:
  redis_cache_index.index_coord(coord)
elif coord.zoom <= 20:
  coord_at_z18 = coord.zoomTo(18).container()
  redis_cache_index.index_coord(coord_at_z18)
```

?